Mobile/Android/Templates/Release_1.9.2/1473-target_marshmallow_rebased.md

```diff
@@ -0,0 +1,22 @@
+###  Marshmallow rebased & copy while renaming files 
+
+#### Pr: https://github.com/owncloud/android/pull/1473 
+
+Devices:
```

Comment:
Should be tested in device with Samsung Galaxy S3 (Android 4, fixes the bug), with HTC (Android 4, changes nothing because they customize the copy/paste overflow menu), and with any Android 5/6 device.